src/lib/holocene/data-encoder-status.svelte

```diff
@@ -16,46 +16,58 @@
   {#if $dataEncoder?.hasNotRequested}
     <Tooltip right text={'Data encoder is configured'}>
```

Comment:
If we use `NavTooltip` here instead then it will follow the same hide/show behavior as the other navigation links.

```suggestion
    <NavTooltip right text={'Data encoder is configured'}>
```

Reply:
I did try this and it introduced some very strange animation behavior that I couldn't pin down. Additionally, I thought that it might be nice to show the tooltip regardless of the nav state because it provides additional context on whether data-encoder is configured.